### PR TITLE
[no bug] Add test suite for browser-tour.js

### PR DESCRIPTION
**Pull request by alexgibson**
Added some base tests for `browser-tour.js`

### CODE REVIEW

File: media/js/test/spec/browser-tour.js [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+        clock.restore();
+    });
+
+    describe('instantiation', function () {
+
+        it('should create a new instance', function() {
+            expect(tour instanceof Mozilla.BrowserTour).toBeTruthy();
+        });
+
+        it('should should call bindEvents', function() {
```
Comment (schalkneethling) [inline media/js/test/spec/browser-tour.js]: double 'should' here

File: media/js/test/spec/browser-tour.js [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+            expect(Mozilla.BrowserTour.prototype.bindEvents).toHaveBeenCalled();
+        });
+
+        it('should should call bindResize', function() {
```
Comment (schalkneethling) [inline media/js/test/spec/browser-tour.js]: another double 'should' here

**Comment by schalkneethling** [comment]
As a follow on from my mail, the code looks good to me but, I am seeing the following test failures:

Firefox 35.0.0 (Mac OS X 10.10) tabzilla.js infobar.update should return true if the user agent is an outdated Firefox version FAILED
	ReferenceError: Tabzilla is not defined in /Users/espressive/repos/bedrock/media/js/test/spec/tabzilla.js (line 146)
	setup@/Users/espressive/repos/bedrock/media/js/test/spec/tabzilla.js:146:17
	@/Users/espressive/repos/bedrock/media/js/test/spec/tabzilla.js:192:20

Is this expected or, is it a config step I am missing?

**Comment by alexgibson** [comment]
Thanks for the r @schalkneethling!

Tabzilla.js is a bit of a special case, as it's actually a view in Python as opposed to a simple JS file, so you need to have bedrock running locally for those tests to pass.

To follow up on your email, currently we are just running these tests locally - but we're looking to get them integrated with Travis soon :+1:

**Comment by schalkneethling** [comment]
well then, r+

**Pull request merged**